user-guide/Advanced_Modules/DOM/DOM_data_storage.md

```diff
@@ -23,3 +29,61 @@ From DataMiner 10.3.9/10.4.0 onwards<!-- RN 36412 -->, a full cache of the DOM c
 >
 > - Correct delivery of the DOM events is important to keep these caches in sync. If the DMS is unstable and Agents frequently lose connection to each other, we recommend disabling the caches. You can do so using the [caching settings](xref:DOM_StorageSettings#cachingsettings) in the `ModuleSettings`.
 > - If a DOM configuration object event is lost, the cache may not reflect the correct state. This will be automatically fixed at midnight, as the "midnight sync" will ensure that the caches are correct by comparing them with the database. You can also trigger this sync manually in the SLNetClientTest tool. For more information, see [Triggering a midnight sync for a DOM manager](xref:SLNetClientTest_triggering_DOM_midnight_sync).
+
+## Cleaning up DOM data from the database
+
+To clean up a DOM manager and remove all DOM data, each DOM object needs to be deleted. This can be done in a custom script using the DOM helper, or via the ["Delete DOM Manager" feature](xref:SLNetClientTest_removing_DOM_Manager) in the DOM window of SLNetClientTest tool. This tool can be used to delete managers with up to 10&thinsp;000 instances. To clean up larger DOM managers, use the script approach first to remove the DOM instances.
+
+> [!NOTE]
+> From DataMiner 10.5.6/10.6.0 onwards<!-- RN TBD -->, it is possible to use the SLNetClientTest tool to clean up larger managers with up to 100&thinsp;000 DOM instances.
```

Comment:
Since there is no release note for this yet, it would be better not to publish this information yet. However, it would be a pity to wait to merge the entire pull request just for this. Would it be OK for you if I set this note in comment for now, so we can merge this?

Reply:
Alright, I would expect this to be available somewhere next week. I'll create another PR then to uncomment it with the right RN.